### ISSUE TITLE
Configure virtualization in your laptop to work with the Status Desktop build

### ISSUE THREAD

**Issue by jorge-campo** [status-im/help.status.im#263]

**Comment by jorge-campo** [comment]
Please, check your box after you complete the tasks in "What to do"

- [ ] Chen
- [ ] Emma
- [ ] Fabio

**Comment by chen0** [comment]
> * [x]  Chen
> * [ ]  Emma
> * [ ]  Fabio

Sure, will wait until my Mac arrives. 

>Installing Windows on a Mac with Apple silicon isn't easy. Hopefully, we can avoid Windows (not sure yet).

Try using cloud services (e.g. AWS EC2) to setup Windows / Linux VMs within minutes, and share the image to others with ease.

**Comment by jorge-campo** [comment]
> Try using cloud services (e.g. AWS EC2) to setup Windows / Linux VMs within minutes, and share the image to others with ease. It avoids newcomers to do any setup work and instantly have the same development environment with the rest of the team.

Good point, @chen0 ! This is something we must investigate, but finding a cloud service that aligns with the organization and getting things ready isn't fast. We need to start somewhere, because we must write docs for desktop OSs now, not in one or two months. Could you please add your proposal as a new backlog project issue in the GitHub project board? Thanks.
